cli/internal/terraform/terraform/aws/main.tf

```
@@ -200,76 +165,16 @@ resource "aws_cloudwatch_log_group" "log_group" {
   tags              = local.tags
 }
 
-module "load_balancer_target_bootstrapper" {
-  source               = "./modules/load_balancer_target"
-  name                 = "${local.name}-bootstrapper"
-  vpc_id               = aws_vpc.vpc.id
-  lb_arn               = aws_lb.front_end.arn
-  port                 = local.ports_bootstrapper
-  tags                 = local.tags
-  healthcheck_protocol = "TCP"
-}
-
-module "load_balancer_target_kubernetes" {
+module "load_balancer_targets" {
+  for_each             = { for port in local.load_balancer_ports : port.name => port }
   source               = "./modules/load_balancer_target"
-  name                 = "${local.name}-kubernetes"
+  name                 = "${local.name}-${each.value.name}"
+  port                 = each.value.port
+  healthcheck_protocol = each.value.health_check
+  healthcheck_path     = each.value.health_check == "HTTPS" ? "/readyz" : ""
```

nit: this seems specific to kubernetes

```suggestion
  healthcheck_path     = each.value.name == "kubernetes" ? "/readyz" : ""
```